packages/components/src/atoms/Slider/Slider.tsx

```diff
@@ -19,7 +19,7 @@ interface RangeLabel {
   max: string | ReactNode
 }
 
-export type SliderProps = {
+export interface SliderProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> {
```

Comment:
is that Omit onChange intentional?

Reply:
Honestly, I don't know. I just converted from what we had before 🙈 
But we don't need this `omit` here since there is no onChange property to omit on this HTML element.

Reply:
I will remove it; it seems to work fine without it 👍